Sources/Crypto/RSA/RSAPadding.swift

```diff
@@ -0,0 +1,49 @@
+import CNIOOpenSSL
+import Foundation
+
+/// RSA Paddings
+public enum RSAPadding: Int32 {
+    /// PKCS #1 v1.5 padding
+    case pkcs1
+    /// SSLv23 padding
+    case sslv23
+    /// No padding
+    case none
+    /// PKCS #1 v2.0 EME-OAEP + SHA-1 + MGF1 padding
+    case pkcs1_OAEP
+    /// X9.31 padding
+    case x931
+
+    public init?(rawValue: Int32) {
```

Comment:
Missing comment. Also should this be a static func named something like `openssl`? Just to be clear it's a raw value from OpenSSL

edit: I guess that wouldn't work if you use raw representable properly, so scratch this.